### ISSUE TITLE
LogcApp connector GA

### ISSUE THREAD

**Issue by kirankumarkolli** [Azure/azure-cosmos-dotnet-v3#3461]

**Comment by NaluTripician** [comment]
Closing as part of a backlog cleanup pass.

**Reason:** Logic Apps connector GA is a separate product, not the .NET SDK. Closing as out of scope for this repo.

**Steps to reopen** - if this is still relevant on a current SDK release:
1. Add a comment with the SDK version you're on, current behavior, and a minimal repro (or updated requirements).
2. Reopen the issue (or ask someone with triage rights to reopen) and the team will re-triage.

cc/auto-triage: closing as `wontfix` per backlog hygiene; reopening is encouraged if circumstances have changed.

**Comment by Pilchie** [comment]
We're still planning this, soon.  @kushagraThapar can you assign someone?

**Comment by kushagraThapar** [comment]
Yes, @NaluTripician will be taking this forward. Assigned it to him!

**Comment by kushagraThapar** [comment]
He might have closed this by mistake as he was trying to clean up all old GitHub issues.

**Comment by NaluTripician** [comment]
Yes @Pilchie  @kushagraThapar, I had some automation going through to close some of the stale GH issues. GA is still on schedule and I am communicating with the Logic Apps team in the final stages.

**Comment by NaluTripician** [comment]
GA released
